asp_interactive_configuration/pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[tool.poetry]
+name = "asp_interactive_configuration"
+version = "0.1.0"
+description = "project to develop an interactive asp-based configuration approach"
+authors = ["gschennersie"]
```

Comment:
According to https://python-poetry.org/docs/pyproject/, authors must be in the form `name <email>`.

Reply:
I don't want to provide my email.